wave_matching/include/wave/matching/matcher.hpp

```diff
@@ -0,0 +1,51 @@
+#ifndef WAVE_MATCHING_MATCHER_HPP
+#define WAVE_MATCHING_MATCHER_HPP
+
+#include <Eigen/Core>
+#include <Eigen/Geometry>
+
+namespace wave {
+
+// sensor data type
+template <typename T>
+class Matcher {
+ public:
```

Comment:
ok mike says nay.

Can you change the `.clang-format` rule from `AccessModifierOffset: -4` to `AccessModifierOffset: -3`